pygis/docs/e_extraction.md

```diff
@@ -0,0 +1,350 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+html_meta:
+  "description lang=en": "Learn how to subset and extract data from a GeoDataFrame through clipping, selecting by attribute, and selecting by location."
+  "keywords": "geospatial, attribute data, subset, extract, selection, vector, shapefile"
+  "property=og:locale": "en_US"
+---
+
+----------------
+
+```{admonition} Learning Objectives
+* Clip a vector feature using another vector feature
```

Comment:
This is good. Keep in mind we are trying to target key search words for SEO both here but primarily in section headings. So try to think of the way you would google it, and use that as the section title